package.json

```diff
@@ -60,14 +60,14 @@
     "karma-firefox-launcher": "^0.1.7",
     "karma-ie-launcher": "^0.2.0",
     "karma-mocha": "^0.2.1",
-    "karma-mocha-reporter": "^1.1.5",
+    "karma-mocha-reporter": "^2.0.0",
     "karma-safari-launcher": "^0.1.1",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
     "lint-staged": "^0.1.1",
     "minimist": "^1.2.0",
     "mocha": "^2.3.4",
-    "ngrok": "0.1.99",
+    "ngrok": "2.1.6",
```

Comment:
Take a look on this issue — https://github.com/bubenshchykov/ngrok/issues/34#issuecomment-195614093
I want to try reinstall node_modules and try again. You made me doubt :D